nil/services/rpc/rawapi/local_txnpool.go

```diff
@@ -25,3 +25,11 @@ func (api *LocalShardApi) SendTransaction(ctx context.Context, encoded []byte) (
 	}
 	return reasons[0], nil
 }
+
+func (api *LocalShardApi) GetTxpoolStatus(ctx context.Context) (uint64, error) {
```

Comment:
I don't really like the name of the method. Why not call it `GetTxpoolQueueueueLen` for example? Seeing the word "status" and some number, I rather expect it to be a value from some enumeration of possible statuses or something.